### PR TITLE
Fix command line option parsing issues #97, #98, #99

### PR DESCRIPTION
**Pull request by andreasstieger**
* account for short/long options #97
* account for multiple and cumulative options #98
* account for quoting styles #99

### CODE REVIEW

**Comment by konstruktoid** [comment]
Sorry for the late reply, but thanks again for your PR @andreasstieger.

**Comment by konstruktoid** [comment]
Got
```sh
[INFO] 2.8  - Do not bind Docker to another IP/Port or a Unix socket
[INFO]      * Docker daemon running with -H
docker-bench-security.sh: 82: ./tests/2_docker_daemon_configuration.sh: get_docker_cumulative_line_args: not found
[INFO] 2.9  - Configure TLS authentication for Docker daemon
```

File: helper_lib.sh [hunk 1]
```diff
@@ -46,3 +46,42 @@ get_command_line_args() {
...
+    # normalize known long options to their short versions
+    sed \
+        -e 's/\-\-debug/-D/g' \
+        -e 's/\-\-host/-H/g' \
+        -e 's/\-\-log-level/-l/g' \
+        -e 's/\-\-version/-v/g' \
+        |
+    # normalize parameters separated by space(s) to -O=VALUE
+    sed \
+        -e 's/\-\([DHlv]\)[= ]\([^- ][^ ]\)/-\1=\2/g' \
```
Comment (andreasstieger) [inline helper_lib.sh]: hmm, not sure about enumerating options here

**Comment by andreasstieger** [comment]
@konstruktoid The two issues are really one, caused by the same typo.

    docker-bench-security.sh: 82: ./tests/2_docker_daemon_configuration.sh: get_docker_cumulative_line_args: not found
    [INFO] 2.9  - Configure TLS authentication for Docker daemon
    [INFO]      * Docker daemon not listening on TCP

Fixed in d2ba1d9f721148eea502791ab8dd9f3199767679, please review.

**Comment by konstruktoid** [comment]
Thanks @andreasstieger, having a look.

**Comment by konstruktoid** [comment]
Thanks alot!

**Pull request merged**